woocommerce/woocommerce-gutenberg-products-block
# Product Collection - Add Display Settings Toolbar

This PR introduces the 'Display Settings' toolbar for the 'Product Collection' block
<img width="412" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/16707866/c98cc9e9-da5f-469b-b323-75c989cd2427">

- A new component 'DisplaySettingsToolbar' is added in 'display-settings-toolbar.tsx'. It includes controls for changing the number of items per page, the offset, and the maximum number of pages to show in the block. The toolbar only shows up when the 'inherit' query attribute is set to false.
- The 'DisplaySettingsToolbar' component is then integrated into the main 'edit.tsx' file through 'ToolbarControls'.
- Styles have also been added to 'editor.scss' to ensure the popover content of the display settings toolbar has an appropriate minimum width.

Fixes #9367


### Testing

1. Add Product Collection block to a post
2. Select the 'Product Collection' block and validate that the 'Display settings' toolbar appears as depicted in the image below:
<img width="412" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/16707866/c98cc9e9-da5f-469b-b323-75c989cd2427">

3. Change 'Items per page', 'Offset', and 'Max page to show'. Verify that these alterations are promptly reflected within the Editor.
4. Publish the post
5. Navigate to the frontend and verify that the changes are accurately reflected there.

Here is quick demo:

https://github.com/woocommerce/woocommerce-blocks/assets/16707866/b8f2a186-d4c5-411a-9210-c40daf00fcf3



### WooCommerce Visibility




### Changelog

> Product Collection - Add Display Settings Toolbar
